host_group_types.go

```diff
@@ -27,3 +27,8 @@ type HostGroup struct {
 	// The hostgroup name.
 	Name string `json:"name,omitempty"`
 }
+
+// Fields returns fields which must be requested to fill struct
+func (h *HostGroup) Fields() []string {
```

Comment:
Can we add a UT for this, I know this is hardcoded function but to not decrease the code coverage.

Reply:
@adarsh-dell code coverage has remained same before and after this fix. Attached are the unit test execution details.
[code_coverage_after_fix.txt](https://github.com/dell/gopowerstore/files/10557262/code_coverage_after_fix.txt)
[code_coverage_before_fix.txt](https://github.com/dell/gopowerstore/files/10557265/code_coverage_before_fix.txt)